src/components/pipeline-run-task-step-log/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { mapRouteParamsToProps } from '../../utils/routing';
+import { Accordion, Typography } from '@equinor/eds-core-react';
+import { usePollLogs } from './use-poll-task-step-logs';
+import { Log } from '../component/log';
+import { AsyncState } from '../../effects/effect-types';
+
+export interface PipelineRunTaskStepLogProps {
+  appName: string;
+  jobName: string;
+  pipelineRunName: string;
+  taskName: string;
+  stepName: string;
+  title: string;
+  logState?: AsyncState<string>;
+}
+
+export const PipelineRunTaskStepLog = ({
+  appName,
+  jobName,
+  pipelineRunName,
+  taskName,
+  stepName,
+  title,
+}: PipelineRunTaskStepLogProps): JSX.Element => {
+  const [logsState] = usePollLogs(
+    appName,
+    jobName,
+    pipelineRunName,
+    taskName,
+    stepName
+  );
+
+  return (
+    <>
+      <Accordion className="accordion elevated" chevronPosition="right">
+        <Accordion.Item isExpanded>
+          <Accordion.Header>
+            <Typography variant="h4">{title}</Typography>
+          </Accordion.Header>
```

Comment:
wrap title in the `Accordion.HeaderTitle` component
```suggestion
          <Accordion.Header>
            <Accordion.HeaderTitle>
                <Typography variant="h4" as="span">{title}</Typography>
            </Accordion.HeaderTitle>
          </Accordion.Header>
```